content/en/docs/setup/quickstart.md

```diff
@@ -3,18 +3,16 @@ title: 'Demo/Evaluation Installations'
 linkTitle: 'Quickstart'
 weight: 1
 description: >
-  Quickstart solutions that gives you the ability to try Spinnaker out quickly. These are not meant for production use as is.
+    Quickstart solutions that gives you the ability to try Spinnaker out quickly. These are not meant for production use as is.
 aliases:
   - /setup/quickstart/
 ---
 
-If you want to install Spinnaker on Lightweight Kubernetes (K3s) for proofs of concept, see the open source project [Minnaker](https://github.com/armory/minnaker). You can install Spinnaker in about 10 minutes in a local or cloud VM.
+There are several simple options to trying spinnaker.  Note that even testing spinnaker does require a fair bit of resources.  Below references are examples only and are not official spinnaker projects but included for ease of use.
 
```

Comment:
I understand the support factor and think the above wording is good.
The `spinnaker-kustomize` repo is the fastest way to get a Spinnaker up and running in my experience.

`$ make create && make build && make apply`
https://github.com/spinnaker/spinnaker-kustomize#quick-start